fem/gridfunc.cpp

```diff
@@ -1581,7 +1581,7 @@ void GridFunction::GetGradient(ElementTransformation &T, Vector &grad) const
          const FiniteElement *fe = fes->GetFE(T.ElementNo);
          MFEM_ASSERT(fe->GetMapType() == FiniteElement::VALUE,
                      "invalid FE map type");
-         MFEM_ASSERT(fes->GetVDim() == 1, "Defined for scalar functions.");
+         MFEM_ASSERT(VectorDim() == 1, "Defined for scalar functions.");
```

Comment:
No need to change this: when the map type is `VALUE` that means that the FE is scalar, `VectorDim()` is the same as `GetVDim()`.